Factorial/src/Factorial.java

```diff
@@ -0,0 +1,80 @@
+import org.junit.jupiter.api.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+import static org.junit.jupiter.api.Assertions.*;
+
+/* Reads an integer and prints n! = 1 * 2 * ... * n to standard output
+   For eg: 0! = 1, 1! = 1, 2! = 2 * 1 = 2, 3! = 3 * 2 * 1 = 6 ..  so on
+   Remarks
+    - There would be an overflow for long if n > 20
+    (Factorial for 21 will give you -> -4249290049419214848 which is incorrect)
+*/
+public class Factorial {
+    // Return n!
```

Comment:
? The name of function and class are descriptive enough, why this comment?

Reply:
Will remove those, had to post that on IG page.
Thanks.